utils/config_manager.py

```
@@ -2233,6 +2593,11 @@ def _cleanup_invalid_workshop_configs(self):
         )
         for candidate in candidates:
             self._cleanup_invalid_workshop_config_file(candidate)
+
+    def repair_workshop_configs(self):
+        """显式修复 workshop 配置文件，仅在调用方明确允许写盘时执行。"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)